src/include/platform/internal/GenericConfigurationManagerImpl.ipp

```diff
@@ -638,17 +638,23 @@ CHIP_ERROR GenericConfigurationManagerImpl<ConfigClass>::IncrementLifetimeCounte
     return mLifetimePersistedCounter.Advance();
 }
 
+template <class ConfigClass>
+CHIP_ERROR GenericConfigurationManagerImpl<ConfigClass>::SetRotatingDeviceIdUniqueId(MutableByteSpan & uniqueIdSpan)
```

Comment:
```suggestion
CHIP_ERROR GenericConfigurationManagerImpl<ConfigClass>::SetRotatingDeviceIdUniqueId(const ByteSpan & uniqueIdSpan)
```
But if it's really supposed to be fixed-length, `FixedByteSpan` might also be appropriate.

Reply:
updated

Reply:
Not setting to FixedByteSpan right now since I can imagine an implementation which allows setting a variable length unique ID (as long as its more than the minimum required).